docs/openapi/api-internal-b2b-webhook.yaml

```diff
@@ -9,27 +9,225 @@ info:
+    - presenterà esclusivamente gli eventi relativi alle notifiche associate ai tali gruppi:
+    - in fase di creazione potranno essere associati i gruppi associati all'api-key utilizzata.
+    - solo le api-key con visibilità sui gruppi dello flusso possano modificarlo, cancellarlo e consumare gli eventi.
+    
+    __NOTA__: Le api-key senza associazione a gruppi possono agire su tutti i gruppi.
+    
+    ## Operazioni sugli stream 
+    
+    Le operazioni sugli stream (tag __Streams__) per la gestione della configurazione:
+      - [Creazione](#/Streams/createEventStream)
+      - [Elenco degli stream configurati](#/Streams/listEventStreams)
+      - [Recupero Configurazione](#/Streams/retrieveEventStream)
+      - [Modifica Configurazione](#/Streams/updateEventStream)
+      - [Cancellazione](#/Streams/removeEventStream) 
+      - [Disabilitazione](#/Streams/disableEventStream)
+    
+    ## Procedura di migrazione agli stream v2.3
+    
+    Per migrare uno stream dalla versione precedente è sufficiente indicare nel parametro _replacedId_ l'identificativo 
+    dello stream precedente in fase di creazione.
+    
+    
+    Il nuovo stream viene creato e contemporaneamente lo stream precedente viene disabilitato.
+    
+    
+    Il mittente deve mantenere attiva la lettura sullo stream precedente fino a consumare tutti gli eventi contenuti e 
+    poi migrare la lettura sul nuovo stream.
+    
+    
+    ## Disabilitazione di uno stream (dalla versione 2.3)
+    
+    Dalla versione 2.3 è possibile disabilitare uno stream tramite la chiamata alla API _disableEventStream_.
+    
+    
+    Uno stream disabilitato non può essere riabilitato, ma è possibile consumare gli eventuali eventi e 
+    sarà eliminato automaticamente dopo <span id="webhookTtlDisabled">14</span> giorni.
+    
+    
+    ## Lettura degli eventi dallo stream
+
+    Le operazioni con il tag __Events__ sono quelle utilizzate per la [lettura degli eventi](#/Events/consumeEventStream) 
+    filtrati in base alla configurazione impostata negli streams.
+    
+    
+    La api restituisce un __massimo di <span id="webhookMaxLength">50</span> elementi per chiamata__. 
+    
+    
+    Se esistono ulteriori eventi nello stream, allora la response del servizio 
+    restituisce l'elemento `retryAfter = 0`; ed è quindi possibile richiamare immediatamente il servizio per ottenere 
+    gli eventi successivi.
+    
+    
+    Per il corretto consumo degli eventi dello stream è necessario passare a tutte le chiamate (ad esclusione della 
+    prima) il parametro _lastEventId_ valorizzato con l'ultimo _eventId_ della richiesta precedente. In questo modo si 
+    informa che gli eventi precedenti sono stati elaborati e quindi possibile procedere con la loro cancellazione dallo 
+    stream e restituire i successivi.
+    
+    
+    Tuttavia, in fase di implementazione del client che consumerà i servizi, bisogna considerare il fattore della 
```

Comment:
Tutto questo paragrafo è un pò "obsoleto", ovvero:
- non c'è più il problema di riproporre eventi anche se molto vicini.
- c'è però il problema di riproporre più volte (causa BUG eh) uno stesso evento inteso non tancome eventId uguale, ma con lo stesso element al suo interno. Di recente per dirti era successo quando abbiam fatto un refactor e spostato l'aggancio degli eventi dello stream sulla lambda piuttosto che inviati dal microservizio stesso. Però oh in futuro chissà cosa riusciam a combinare....

non so se ha senso citare la cosa o se è meglio togliere del tutto.